catalog/resourcemanagement/resourcemanagement-querymonitor-plugin/pom.xml

```diff
@@ -17,12 +17,12 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <artifactId>plugin</artifactId>
-        <groupId>ddf.catalog.plugin</groupId>
+        <groupId>org.codice.ddf.resourcemanagement</groupId>
+        <artifactId>resourcemanagement</artifactId>
         <version>2.9.0-SNAPSHOT</version>
     </parent>
-    <artifactId>catalog-plugin-querymonitor</artifactId>
-    <name>DDF :: Catalog :: Plugin :: Query Monitor Service</name>
+    <artifactId>resourcemanagement-querymonitor-plugin</artifactId>
+    <name>DDF :: Resource Management :: Query Monitor Plugin</name>
```

Comment:
We have a resource management app.  https://github.com/codice/ddf/tree/master/catalog/resourcemanagement